website/docs/r/rde_type.html.markdown

```diff
@@ -85,6 +85,41 @@ resource "vcd_rde_type" "my_rde_type" {
 }
 ```
 
+## Example Usage with Interface Behaviors and Hooks
+
+```hcl
+data "vcd_rde_interface" "my_interface" {
+  vendor  = "bigcorp"
+  ns      = "tech1"
+  version = "1.2.3"
+}
+
+resource "vcd_rde_interface_behavior" "my_behavior" {
+  interface_id = vcd_rde_interface.my_interface.id
+  name         = "MyBehavior"
+  description  = "Adds a node to the cluster.\nParameters:\n  clusterId: the ID of the cluster\n  node: The node address\n"
```

Comment:
This description seems out of place?

Reply:
Actually I got this description from [the API docs](https://developer.vmware.com/apis/vmware-cloud-director/latest/cloudapi/1.0.0/interfaces/id/behaviors/post/), as it is probably taken from a real Behavior, rather putting a made up one that could be misleading (ideally the description should define the contract of the Behavior).

But I can change it to something less specific, like 
```
"Calls an example  Behavior.\nParameters:\n  parameter1: the first param\n  parameter2: the second param\n"
```
that way it would look less odd.

Reply:
A! In the API docs it looks a bit less peculiar, as other fields contain information about "containerCluster". Here, we don't, so it feels better to have a generic description.